src/geometry/manifold/instances/units_of_normed_algebra.lean

```diff
@@ -0,0 +1,40 @@
+/-
+Copyright © 2021 Heather Macbeth, Nicolò Cavalleri. All rights reserved.
+Released under Apache 2.0 license as described in the file LICENSE.
+Authors: Heather Macbeth, Nicolò Cavalleri
```

Comment:
Ok for the copyright. I think you should be the first authors because you made most of the design choices and the rule was that authors are those who should be able to explain the design choices.

Reply:
This is rather the rule for adding subsequent authors -- the first author is the person who literally wrote the first version of the file.

Reply:
Ok